spec/index.html

```diff
@@ -283,9 +283,157 @@ <h2>Introduction</h2>
+      of linked data documents in YAML, but even to define the expected behavior of
+      applications that processes these documents, including fragment identifiers and
+      interoperability consideration.
+
+      This is because YAML is more flexible than JSON:
+
+      * it supports different encodings, including UTF-8, UTF-16 and UTF-32;
+      * it supports more data types than JSON;
+      * the structure of its documents, that is named YAML representation graph,
+        is a tree that can have cycles.
+    </p>
+
+    <p>
+      The first goal of this specification is to allow to process a JSON-LD document
+      and serialize it in YAML, and then back to JSON-LD without losing any information.
+
+      This is always possible, because a YAML representation graph can always represent
+      a non-cyclic tree, because JSON data types are a subset of YAML's, and because
+      JSON encoding is UTF-8.
+    </p>
+
+    <p data-format="markdown>Example: the JSON-LD document below
+
+    ```
+    {
+      "@context": "http://example.org/context.jsonld",
+      "@graph": [
+        {"@id": "http://example.org/1", "title": "Example 1"},
+        {"@id": "http://example.org/2", "title": "Example 2"},
+        {"@id": "http://example.org/3", "title": "Example 3"}
+      ]
+    }
+    ```
+
+    can be serialized as YAML as follows. Note that - since `@`
+    is a reserved character in YAML, entries containing it 
+    needs to be enclosed in double quotes.
+
+    ```yaml
+    %YAML 1.2
+    ---
+    "@context": http://example.org/context.jsonld
+    "@graph":
+      -
+        "@id": http://example.org/1
+        title: Example 1
+      -
+        "@id": http://example.org/2
+        title: Example 2
+      -
+        "@id": http://example.org/3
+        title: Example 3
+    ```
+
+
+    </p>
   </section>
+  <section id="specifications" class="normative">
+    <h2>Core Requirements</h2>
+
+    <p>
+      A YAML-LD document is a [[YAML]] document that can be interpreted as linked data.
+    </p>
+    <p>
+      It MUST be encoded in UTF-8, to improve interoperability with [[JSON]]. 
+    </p>
+    <p>
+      Comments in YAML-LD documents
+      MAY be removed when interpreting the document
+      as JSON-LD.
+      It is then legitimate for a YAML-LD implementation to ignore comments,
+      and relevant information SHOULD be provided using specific predicates
+      such as rdfs:comment.
+      This specification does not provide a means for preserving
+      YAML comments after a JSON serialization.
+      See Interoperability considerations of [[I-D.ietf-httpapi-yaml-mediatypes]] for more details.
+    </p>
+    <p>
+      Since named anchors are a serialization detail,
+      such names
+      MUST NOT be used to convey relevant information,
+      MAY be altered when processing the document,
+      and MAY not be persisted when interpreting the document as JSON-LD.
+
+    </p>
```

Comment:
Probably should provide a reference for "named anchors", clarify that they are part of an _expanded profile_, and that when transforming to the _internal representation_, references are expanded in place. This contrasts with your **MUST NOT** statement, but I would expect a YAML de-serializer to do the heavy lifting for this, while removing any notion of those anchors and references in the post-processed form.

Reply:
Okay, much of what I said is described below, but  maybe the paragraph needs to be tweaked to allow for this.

Reply:
We also need to clarify that these features are **NOT** supported in the base profile, only in the extended profile.

Reply:
@gkellogg I am opening a separate issue for discussing profiles: https://github.com/json-ld/yaml-ld/issues/35

My proposal for now is to retain the following information, and move them around when fixing #35 

WDYT?

Reply:
@gkellogg if that's ok, just resolve this conversation. This comment should be already linked in #35